source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,188 @@
+
+Removing All Secure Connections from a Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within a specific channel, use the following slash command to remove all secure connections: 
+
+``/share-channel unshare``
+
+This slash command removes all secure connections from the current channel. Secure connections may continue to be invited to other shared channels. 
+
+While unsharing a shared channel stops synchronizing the channel with the other Mattermost server, the channel continues to function for local users as expected. 
+
+.. note:: 
+    A System message notifies System Admins that the channel is no longer shared.
+
+Reviewing Secure Connections in Channels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following slash command to review the status of all secure connections within the current shared channel:
+
+``/share-channel status``
+
+Status details include:
+- Connection ID
+- Connection URL
+- Description
+- Read only channel (True/False)
+- Invite accepted (Yes/No)
+- Online (Yes/No)
+- Last ping timestamp (UTC)
+
+Frequently Asked Questions
+---------------------------
+
+Are special characters supported in secure connection names?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No. ``--name`` can include periods, hyphens, and/or underscores. You must surround ``--name`` and ``--displayname`` values using quotation marks (“ “) when the values contain spaces.
+
+What happens if two Mattermost instances contain different emojis?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In cases where one Mattermost instance has different emojis than another instance, emoji text displays in place of a missing emoji image. 
+
+Is a Display Name required for all secure connections?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No. ``--displayname`` is optional. When omitted, ``--name`` is displayed and used instead.
+
+Do connection interruptions affect message synchronization?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. Both System Admins and users are notified when a connection interruption is preventing message synchronization. System Admins and users are also notified connections are restored between shared Mattermost instances.
+
+What happens if two secure connections share the same usernames?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In cases where members share the same usernames across Mattermost secure connections, usernames on the local server instance are appended with the secure connection ID of the remote server.
+
+For example, if multiple members named John Smith exist after two Mattermost instances establish a secure connection with one another, all remote John Smith members include their Secure Connection ID following their username to help differentiate members across multiple Mattermost instances.
```

Comment:
Why is this feature in Beta? 

This feature is in beta while we recruit customer testing partners.  This feature has been tested by Mattermost QA, but as we build the UI for managing shared channels, we want to work with administrators to build the most optimal experience.

Reply:
Great addition! Thank you!